layout/js/map.js

```diff
@@ -185,11 +185,12 @@ var get_data = function() {
     transformData = data;
 
     var slider = document.getElementById('slider');
-    slider.noUiSlider.on('update', function( values, handle ) {
-  	  var year = "" + Math.round(values[handle]);
-    	setYear(year);
-    });
-
+    if(slider.noUiSlider){
```

Comment:
This is right, the timing issue is because we have two onloads that can conflict.  We should probably put some promises in to fix the dependency here, but for now this is good.  We should add an else with a `setTimeout` so the slider eventually gets loaded.